index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>    
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <h2><a href="#">Welcome</a></h2>
+        <i class="fa fa-bars" aria-hidden="true"></i>
+    </header>
+    <section class="home-page">
+        <h1 class="intro">Hey there. I am Selma <br><span>I am a software<br> developer</span></h1>
```

Comment:
- [ ] To be more in line with semantic HTML here, it'd be better if "I am a software developer" was instead on an h2 tag below, it's ok for both of them to have the same size, you can change by adding a class of "sub-intro", the reason is that the h1 tag has implications in the accessibility of the page for screen readers, and "I am a software developer" is better treated as a subtitle